src/components/HOC/WithProfile/WithProfile.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react'
+import NotFoundPage from '../../Pages/NotFoundPage'
+import { Props } from './WithProfile.types'
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const WithProfile = (props: Props) => {
+  const { component: Component, error, isAddress, isLoading, hasLoadedProfile, addressOrName, profileAddress, onFetchProfile } = props
```

Comment:
Having to capitalize custom components is a weird thing from react

<img width="890" alt="image" src="https://github.com/decentraland/profile/assets/24811313/0dfdf8d2-0e4a-47fc-ba44-e209f944aad1">

Reply:
Has been like that for forever it seems https://legacy.reactjs.org/blog/2015/09/10/react-v0.14-rc1.html

Reply:
At least when using TS, in JSX, the built-in component types are looked for in a uncapitalized way. I imagine this is because the case sensitive nature of JS or TS itself.